site/content/docs/manifest/worker-service.ja.md

```diff
@@ -0,0 +1,101 @@
+以下は `'Worker Service'` Manifest で利用できるすべてのプロパティのリストです。[Service の概念](../concepts/services.ja.md)説明のページも合わせてご覧ください。
+
+???+ note "Worker Service の Manifest のサンプル"
+
+    ```yaml
+    # Service 名は、ロググループや ECS サービスなどのリソースの命名に使用されます。
+    name: orders-worker
+    type: Worker Service
+
+    # コンテナや Service の設定
+    image:
+      build: ./orders/Dockerfile
+
+    subscribe:
+      topics:
+        - name: events
+          service: api
+        - name: events
+          service: fe
+      queue:
+        retention: 96h
+        timeout: 30s
+        dead_letter:
+          tries: 10
+
+    cpu: 256
+    memory: 512
+    count: 1
+
+    variables:
+      LOG_LEVEL: info
+    secrets:
+      GITHUB_TOKEN: GITHUB_TOKEN
+
+    # 上記で定義された値は、environment で上書きすることができます。
+    environments:
+      test:
+        count:
+          spot: 2
+      production:
+        count: 2
+    ```
+
+<a id="name" href="#name" class="field">`name`</a> <span class="type">String</span>
+Service の名前。
+
+<div class="separator"></div>
+
+<a id="type" href="#type" class="field">`type`</a> <span class="type">String</span>  
+Service のアーキテクチャタイプ。[Worker Service](../concepts/services.ja.md#worker-service) は、インターネットや VPC 外からはアクセスできません。これは、他のCopilot Service の `publish` フィールドで作成された SNS トピックへのサブスクリプションによって生成されます。
```

Comment:
I think something is missing in the translated text?

This is the original version:
```
The architecture type for your service. [Worker Services](../concepts/services.en.md#worker-service) are not reachable from the internet or elsewhere in the VPC. They are designed to pull messages from their associated SQS queues, which are populated by their subscriptions to SNS topics created by other Copilot services' `publish` fields.
```

Reply:
It's true!
I'll confirm and add it.